.github/workflows/gha-deploy-integration.yaml

```diff
@@ -82,24 +83,28 @@ jobs:
           tags: |
             type=sha,enable=true,priority=600,prefix=gha-,suffix=
             type=raw,enable=true,priority=600,prefix=gha-deploy-int-,value=${{ steps.timestamp.outputs.date }}
-
-      - uses: int128/kaniko-action@v1.43.0
-        name: Build with Kaniko
-        with:
-          push: true
-          file: "${{ inputs.DOCKERFILE }}"
-          tags: ${{ steps.metadata.outputs.tags }}
-          labels: ${{ steps.metadata.outputs.labels }}
-          cache: ${{ inputs.KANIKO_CACHE }}
-          cache-repository: ${{ inputs.AWS_ACCOUNT_ID }}.dkr.ecr.${{ inputs.AWS_REGION }}.amazonaws.com/kaniko-cache
```

Comment:
Isn't this still needed to actually build the image?

Reply:
My bad removed by mistake